{
    "timestamp": "2025-04-12 11:38:48",
    "client_ip": "127.0.0.1",
    "method": "POST",
    "url": "http://localhost:8000/posts/",
    "headers": {
        "host": "localhost:8000",
        "connection": "keep-alive",
        "content-length": "377",
        "sec-ch-ua-platform": "\"Windows\"",
        "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 YaBrowser/25.2.0.0 Safari/537.36",
        "accept": "application/json",
        "sec-ch-ua": "\"Not A(Brand\";v=\"8\", \"Chromium\";v=\"132\", \"YaBrowser\";v=\"25.2\", \"Yowser\";v=\"2.5\"",
        "content-type": "application/json",
        "sec-ch-ua-mobile": "?0",
        "origin": "http://localhost:8000",
        "sec-fetch-site": "same-origin",
        "sec-fetch-mode": "cors",
        "sec-fetch-dest": "empty",
        "referer": "http://localhost:8000/docs",
        "accept-encoding": "gzip, deflate, br, zstd",
        "accept-language": "ru,en;q=0.9"
    },
    "error_type": "ProgrammingError",
    "error_message": "(psycopg2.errors.UndefinedColumn) column user_table.document_scores_history does not exist\nLINE 1: ...cription, user_table.rating AS user_table_rating, user_table...\n                                                             ^\n\n[SQL: SELECT user_table.user_id AS user_table_user_id, user_table.login AS user_table_login, user_table.password AS user_table_password, user_table.type_id AS user_table_type_id, user_table.name AS user_table_name, user_table.image_link AS user_table_image_link, user_table.description AS user_table_description, user_table.rating AS user_table_rating, user_table.document_scores_history AS user_table_document_scores_history \nFROM user_table \nWHERE user_table.user_id = %(user_id_1)s \n LIMIT %(param_1)s]\n[parameters: {'user_id_1': 3, 'param_1': 1}]\n(Background on this error at: https://sqlalche.me/e/20/f405)",
    "traceback": "Traceback (most recent call last):\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 1964, in _exec_single_context\n    self.dialect.do_execute(\n    ~~~~~~~~~~~~~~~~~~~~~~~^\n        cursor, str_statement, effective_parameters, context\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\default.py\", line 945, in do_execute\n    cursor.execute(statement, parameters)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\npsycopg2.errors.UndefinedColumn: column user_table.document_scores_history does not exist\nLINE 1: ...cription, user_table.rating AS user_table_rating, user_table...\n                                                             ^\n\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 214, in run_endpoint_function\n    return await run_in_threadpool(dependant.call, **values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\starlette\\concurrency.py\", line 37, in run_in_threadpool\n    return await anyio.to_thread.run_sync(func)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\anyio\\to_thread.py\", line 56, in run_sync\n    return await get_async_backend().run_sync_in_worker_thread(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n        func, args, abandon_on_cancel=abandon_on_cancel, limiter=limiter\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 2470, in run_sync_in_worker_thread\n    return await future\n           ^^^^^^^^^^^^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 967, in run\n    result = context.run(func, *args)\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\app\\routes.py\", line 94, in create_post\n    return PostService.create_post(db=db, post_data=post)\n           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\app\\services\\post_service.py\", line 23, in create_post\n    user = db.query(User).filter(User.user_id == post_data.user_id).first()\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\orm\\query.py\", line 2759, in first\n    return self.limit(1)._iter().first()  # type: ignore\n           ~~~~~~~~~~~~~~~~~~~^^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\orm\\query.py\", line 2858, in _iter\n    result: Union[ScalarResult[_T], Result[_T]] = self.session.execute(\n                                                  ~~~~~~~~~~~~~~~~~~~~^\n        statement,\n        ^^^^^^^^^^\n        params,\n        ^^^^^^^\n        execution_options={\"_sa_orm_load_options\": self.load_options},\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\orm\\session.py\", line 2365, in execute\n    return self._execute_internal(\n           ~~~~~~~~~~~~~~~~~~~~~~^\n        statement,\n        ^^^^^^^^^^\n    ...<4 lines>...\n        _add_event=_add_event,\n        ^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\orm\\session.py\", line 2251, in _execute_internal\n    result: Result[Any] = compile_state_cls.orm_execute_statement(\n                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^\n        self,\n        ^^^^^\n    ...<4 lines>...\n        conn,\n        ^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\orm\\context.py\", line 306, in orm_execute_statement\n    result = conn.execute(\n        statement, params or {}, execution_options=execution_options\n    )\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 1416, in execute\n    return meth(\n        self,\n        distilled_parameters,\n        execution_options or NO_OPTIONS,\n    )\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 523, in _execute_on_connection\n    return connection._execute_clauseelement(\n           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^\n        self, distilled_params, execution_options\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 1638, in _execute_clauseelement\n    ret = self._execute_context(\n        dialect,\n    ...<8 lines>...\n        cache_hit=cache_hit,\n    )\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 1843, in _execute_context\n    return self._exec_single_context(\n           ~~~~~~~~~~~~~~~~~~~~~~~~~^\n        dialect, context, statement, parameters\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 1983, in _exec_single_context\n    self._handle_dbapi_exception(\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^\n        e, str_statement, effective_parameters, cursor, context\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 2352, in _handle_dbapi_exception\n    raise sqlalchemy_exception.with_traceback(exc_info[2]) from e\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\base.py\", line 1964, in _exec_single_context\n    self.dialect.do_execute(\n    ~~~~~~~~~~~~~~~~~~~~~~~^\n        cursor, str_statement, effective_parameters, context\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\\u041f\u0440\u043e\u0435\u043a\u0442\u044b\\Threads_Hack\\venv\\Lib\\site-packages\\sqlalchemy\\engine\\default.py\", line 945, in do_execute\n    cursor.execute(statement, parameters)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\nsqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedColumn) column user_table.document_scores_history does not exist\nLINE 1: ...cription, user_table.rating AS user_table_rating, user_table...\n                                                             ^\n\n[SQL: SELECT user_table.user_id AS user_table_user_id, user_table.login AS user_table_login, user_table.password AS user_table_password, user_table.type_id AS user_table_type_id, user_table.name AS user_table_name, user_table.image_link AS user_table_image_link, user_table.description AS user_table_description, user_table.rating AS user_table_rating, user_table.document_scores_history AS user_table_document_scores_history \nFROM user_table \nWHERE user_table.user_id = %(user_id_1)s \n LIMIT %(param_1)s]\n[parameters: {'user_id_1': 3, 'param_1': 1}]\n(Background on this error at: https://sqlalche.me/e/20/f405)\n"
}